contentcuration/contentcuration/frontend/settings/views/Storage/index.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+
+  <div>
+    <p>
+      <h2>{{ storageUsagePercentage }}% storage used</h2>
+      <KLinearLoader :progress="storageUsagePercentage" type="determinate" class="loader" />
+      <div>{{ `${bytesToHuman(usedSpace)} GB of ${bytesToHuman(totalSpace)} GB` }}</div>
+    </p>
+
+    <KFixedGrid numCols="8" gutter="10">
+      <template v-for="kind in kindKeys">
+        <KFixedGridItem span="2" class='row' :key="`${kind}1`">
+          <span :style="{ backgroundColor: kindColors(kind), padding: '8px', marginRight: '8px' }">
+            <KIcon :icon="kind" :color="$themeTokens.textInverted" />
+          </span>
+          <span>{{ kindLabel(kind) }}</span>
+        </KFixedGridItem>
+        <KFixedGridItem span="6" class='row' :key="`${kind}2`">
+          <span>{{ `${size(kind)} GB` }}</span>
+        </KFixedGridItem>
+      </template>
+    </KFixedGrid>
+
+    <p>
+      <h2>Request more space</h2>
+      <p>
+        <span>Kindly use this form to request additional uploading storage for your Kolibri Studio account. If you import content from our public library in your channels, this content does not count towards your storage limit. <KButton appearance="basic-link" style="display: inline;" text="Learn more how to import content from other channels." @click="() => console.log('toimplement')" /></span>
+      </p>
+      <KButton appearance="basic-link" :text="showRequestForm ? 'Hide form' : 'Show form'" @click="showRequestForm = !showRequestForm" />
+    </p>
+    <RequestForm v-if="showRequestForm" />
+  </div>
+
+</template>
+
+
+<script>
+
+import { mapState } from 'vuex';
+import RequestForm from './RequestForm';
+
+const kindKeys = [
```

Comment:
There is a [constants file](https://github.com/learningequality/studio/blob/vue-refactor/contentcuration/contentcuration/frontend/shared/leUtils/ContentKinds.js) available so you don't have to create a new one. You can also use the ConstantsTranslationMixin to translate the kinds with `translateConstant`